src/documents/tutorial-prefsFrameworkMinimalFootprint/MinimalFootprint.md

```diff
@@ -0,0 +1,133 @@
+
+<div class="infusion-docs-note"><strong>Note:</strong> Preferences Editors can be configured either through an <a href="../AuxiliarySchemaForPreferencesFramework.md">Auxiliary Schema</a> or directly through grades. The grade version is slightly more complicated to configure, but removes computation time for processing the schema.</div>
+
+#### Instantiating a Full Page Preferences Editor ####
+
+The following example instantiates a Full Page Preferences Editor with a preview window.
+
+```javascript
+    /*
+     * The various starter gradeNames mentioned below indicate that the
+     * "starter" adjusters and enactors should be used. These correspond to the
+     * same set of adjusters and enactors used in a typical UI Options
+     * configuration. However, a custom set of adjusters and enactors could be
+     * configured here.
+     */
+
+    /**
+     * Initialize PrefsEditor global settings store.
+     */
+    fluid.prefs.globalSettingsStore();
+
+    /**
+     * Initialize UI Enhancer for the page.
+     */
+    fluid.pageEnhancer({
+        uiEnhancer: {
+            gradeNames: ["fluid.uiEnhancer.starterEnactors"],
+            tocTemplate: "path/to/TocTemplate"
+        }
+    });
+
+    /**
+     * Initialize Full Page preferences editor
+     */
+    fluid.prefs.fullPreview(".container", {
+        gradeNames: ["fluid.prefs.initialModel.starter"],
+        // Tell PrefsEditor where to find all the templates, relative to this
+        // file
+        terms: {
+            templatePrefix: "path/to/templates",
+            messagePrefix: "path/to/messages"
+        },
+        messageLoader: {
+            gradeNames: ["fluid.prefs.starterMessageLoader"]
+        },
+        prefsEditor: {
+            gradeNames: [
+                "fluid.prefs.starterPanels",
+                "fluid.prefs.uiEnhancerRelay"
+            ]
+        },
+        templateLoader: {
+            gradeNames: ["fluid.prefs.starterFullPreviewTemplateLoader"]
+        }
+    });
+```
+
+### Lazy Loading ###
+
+With a Full Page Preferences Editor we've moved editing preferences off of each of the content pages; however, these pages are still required to enact/apply the preferences set by the user. Therefore, we still need to instantiate the [Settings Store](../SettingsStore.md) and [Page Enhancer](../Enactors.md) on each page. However, we only need these if a user has actually adjusted their preferences. In a default implementation of the Preferences Framework, preferences are stored in a browser cookie. The simple solution is to check for the presence of this cookie, and only load and instantiate the Settings Store and Page Enhancer if it's found.
+
+The following example makes use the functions provided by [loadScripts.js](../examples/loadScripts.js) to check the cookie and lazy load the required scripts:
```

Comment:
Also, requiring this be moved into Infusion means that this PR is blocking the Infusion 2.0 release and not just the Docs. So we'll need to decide, implement, and test this as soon as possible.